src/applications/disability-benefits/686c-674-v2/config/chapters/report-add-child/additionalInformationPartTwo.js

```diff
@@ -0,0 +1,30 @@
+import {
+  titleUI,
+  radioUI,
+  radioSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+export const additionalInformationPartTwo = {
+  uiSchema: {
+    ...titleUI('Additional information about this child'),
+    incomeInLastYear: radioUI({
+      title:
+        'Did this child have income in the last 365 days? Answer this question only if you are adding this dependent to your pension.',
+      required: () => true,
+      labels: {
+        Y: 'Yes',
+        N: 'No',
+      },
+      errorMessages: {
+        required: 'Please select an option.',
+      },
+    }),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      incomeInLastYear: radioSchema(['Y', 'N']),
+    },
+    required: ['incomeInLastYear'],
+  },
```

Comment:
We can use `yesNoUI` and `yesNoSchema` for these booleans. Also we shouldn't set required twice. Since it is set in the uiSchema you can remove it from the schema just in case to prevent any submission issues